docs/sources/integrations/servicenow/index.md

```diff
@@ -87,31 +107,31 @@ Fill in the following details:
 
 ## Connect other integrations
 
-You can connect other integrations such as Alertmanager, Grafana Alerting, and others to an existing ServiceNow
-integration. When connected, Grafana OnCall will send alerts from the connected integrations to ServiceNow, and update
-alert groups on the connected integrations based on incident state changes in ServiceNow. Connected integrations will
-use the same ServiceNow credentials and outgoing webhooks as the ServiceNow integration they are connected to.
+You can link other integrations such as Alertmanager, Grafana Alerting, and others to your ServiceNow integration for a consolidated workflow.
```

Comment:
I think using "connect" here would make things more consistent, since the UI always refers to this as "connect". My concern is that users could get the wrong idea that there are two different operations "link" vs "connect"

Reply:
That's a good point - I think connect does make more sense. Let's swap those out.